remc2/engine/engine_support.cpp

```diff
@@ -857,15 +859,15 @@ uint32_t compare_with_sequence_E7EE0(char* filename, uint8_t* adress, uint32_t
 };
 
 uint32_t compare_with_sequence_D41A0(char* filename, uint8_t* adress, uint32_t  /*adressdos*/, uint32_t count, uint32_t size, uint8_t* origbyte, uint8_t* copybyte, long offset, bool regressions) {
-	char findnamec[512];
+	std::string findname;
 	char findnamec2[512];

```

Comment:
We need to do more of this. So many sprintf's in the code. One day I will get to them with the all the path stuff